### PR TITLE
Add SYNCEXTRA environment variable

### PR DESCRIPTION
**Pull request by sven-codeculture**
Make it possible to add extra parameters to the sync_cmd (such as --cache-control, for creating cache headers for all uploaded files)

This would be really helpful since i sync an CDN using this project

### CODE REVIEW

File: s3sync/entrypoint.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ AWS_S3_SSE="${AWS_S3_SSE:-false}"
 AWS_S3_SSE_KMS_KEY_ID="${AWS_S3_SSE_KMS_KEY_ID:-}"
 CRON_TIME="${CRON_TIME:-10 * * * *}"
 INITIAL_DOWNLOAD="${INITIAL_DOWNLOAD:-true}"
+SYNCEXTRA=${SYNCEXTRA:-}
```
Comment (vladgh) [inline s3sync/entrypoint.sh]: Would you please quote this (double quotes)

File: s3sync/entrypoint.sh [hunk 2]
```diff
@@ -24,7 +25,7 @@ sync_files(){
   src="${1:-}"
   dst="${2:-}"
 
-  sync_cmd='--no-progress --delete --exact-timestamps'
+  sync_cmd='--no-progress --delete --exact-timestamps '$SYNCEXTRA
```
Comment (vladgh) [inline s3sync/entrypoint.sh]: This also needs to be changed to double quotes, with $SYNCEXTRA included.

**Comment by vladgh** [comment]
Let me know how it works in production.

**Comment by sven-codeculture** [comment]
I am already using it in Production. I've currently replaced the file in an own docker image based on yours.
By adding SYNCEXTRA: --cache-control max-age=2419200,public to my docker-compose for the sync command, every newly uploaded file gets the header "cache-control: max-age=2419200,public" which i require for pagespeed on the CDN.
Unfortunatly it only works for newly uploaded files, already existing files needs to be modifyed manually.

**Pull request merged**